packages/remix-server-runtime/rollup.config.js

```diff
@@ -14,15 +13,6 @@ const {
 } = require("../../rollup.utils");
 const { name: packageName, version } = require("./package.json");
 
-const ENABLE_REMIX_ROUTER = !!process.env.ENABLE_REMIX_ROUTER;
-
-const replacePlugin = replace({
-  preventAssignment: true,
-  values: {
-    "process.env.ENABLE_REMIX_ROUTER": ENABLE_REMIX_ROUTER ? "1" : "0",
-  },
-});
```

Comment:
Revert the `rollup.config.js` changes from https://github.com/remix-run/remix/pull/4245